[coinbase-v2.2.0-alpha.1]: Upgrade coinbase wallet sdk

cb-jake opened the pull request:
### Description
upgrade coinbase wallet sdk v3.6.0

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

Adamj1232 commented on packages/coinbase/yarn.lock:
```diff
@@ -0,0 +1,1968 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
@cb-jake can you please remove this yarn.lock ad it is not required

Adamj1232 commented on packages/coinbase/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/coinbase",
-  "version": "2.1.3",
+  "version": "2.2.0",
```
```suggestion
  "version": "2.2.0-alpha.1",
```
This will allow for publishing of an alpha package for testing

Adamj1232 reviewed (changes requested):
@cb-jake thanks for the PR! a couple small comment. Can you also remove the packages/coinbase/yarn.lock file as its not required Thanks!

cb-jake commented:
Thanks @Adamj1232. I addressed your comments in [5615ce9](https://github.com/blocknative/web3-onboard/pull/1373/commits/5615ce911dd544ce5e0375bdf62916a718df28f9)

Adamj1232 commented:
@cb-jake we had a PR merged to fix the entry point for the coinbase SDK in certain envs. If you could please pull in the latest develop and push we can get this PR merged tomorrow. Thank you!

cb-jake commented:
Thanks @Adamj1232! I just rebased. LMK if anything else is needed!

Adamj1232 commented on packages/coinbase/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/coinbase",
-  "version": "2.1.4-alpha.1",
+  "version": "2.2.1-alpha.1",
```
```suggestion
  "version": "2.2.0-alpha.1",
```

Adamj1232 commented:
> Thanks @Adamj1232! I just rebased. LMK if anything else is needed!

Great one small note on the PR. We will try and get an alpha version out this afternoon. Thanks again!

Adamj1232 reviewed (approved):
Thank you for this update, its much appreciated!!

cb-jake commented:
Thanks again @Adamj1232!
